src/components/tech/HeroSection.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from "next/image";
+import { Box, Flex, Heading, Text, Button } from "@radix-ui/themes";
+import heroBackgroundImg from "../../../public/images/tech/tech-hero-background-img.jpg";
+const HeroSection = () => {
+  return (
+    <div className="block w-[1252px] item-center justify-center p-8 relative ">
```

Comment:
**Suggestion (non-blocking):** Would be good to use RadixUI's [Flex Component](https://www.radix-ui.com/themes/docs/components/flex) if possible.

**Question:** Why are we using such a specific width?